envs/monkey_zoo/blackbox/test_blackbox.py

```diff
@@ -130,15 +129,6 @@ def test_depth_4_a(self, island_client):
             island_client, depth_4_a_test_configuration, "Depth4A test suite"
         )
 
-    # Not grouped because can only be ran on windows
-    @pytest.mark.skip_powershell_reuse
```

Comment:
You need to remove this mark from `monkey/_zooblackbox/conftest.py`

Reply:
Why was this even here? Do I need to re-introduce it?